.github/workflows/beep-lint.yml

```diff
@@ -0,0 +1,37 @@
+name: Linting
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.8]
```

Comment:
I doubt different python versions will change linting, but for symmetry, would keeping this at the max version used in testing and deployment make sense?